src/test/scala/com/microsoft/hyperspace/index/E2EHyperspaceRulesTests.scala

```diff
@@ -495,13 +570,12 @@ class E2EHyperspaceRulesTests extends HyperspaceSuite with SQLHelper {
     }.flatten
   }
 
-  private def getIndexFilesPath(indexName: String, version: Int = 0): Seq[Path] = {
-    Content
-      .fromDirectory(
-        new Path(
-          systemPath,
-          s"$indexName/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=$version"))
-      .files
+  private def getIndexFilesPath(indexName: String, version: Seq[Int] = Seq(0)): Seq[Path] = {
+    version.flatMap(v =>
```

Comment:
nit: usually, `{` is preferred for multi lines:
```scala
    versions.flatMap { v =>
      Content
        .fromDirectory(
          new Path(systemPath, s"$indexName/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=$v"))
        .files
    }
```

Reply:
thanks @imback82 , added